Contributing.md

```diff
@@ -0,0 +1,61 @@
+Contributor Guidelines
+===
+
+## Code Style
+* Line endings
```

Comment:
"We generally follow the [Google style guidelines](https://google.github.io/styleguide/javaguide.html), but some of the more important things to note are:"